sympy/polys/domains/pythonintegerring.py

```diff
@@ -29,7 +29,11 @@ def to_sympy(self, a):
 
     def from_sympy(self, a):
         """Convert SymPy's Integer to ``dtype``. """
-        if a.is_Integer:
+        if isinstance(a, PythonInteger):
```

Comment:
This method is intended for converting sympified integers to Python int's. It should not be used for `PythonInteger`s which are not SymPy objects.